sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 3.16.0 (2022-05-20)
```

Comment:
Can you please update the date before merging?

Reply:
done